test/bin/mdx-test/expect/simple-mli/test-case.mli

```diff
@@ -0,0 +1,20 @@
+(** This is a doc comment with some code blocks in it:
+
+    {[
+      # List.map (fun x -> x * x) [(1 + 9); 2; 3]
+      - : int list = [100; 4; 9]
+    ]}
+
+    {[List.map (fun x -> x * x) [1; 2; 3]]}
```

Comment:
Could you add a multi line code block that is not toplevel ? Currently the `pp_lines` function is not tested.

Reply:
Now that this has merged, I'll address this and your other pending comment in another pull request. Thanks!

Reply:
https://github.com/realworldocaml/mdx/pull/242